ultralytics/engine/exporter.py

```diff
@@ -604,6 +611,28 @@ def export_ncnn(self, prefix=colorstr("NCNN:")):
         yaml_save(f / "metadata.yaml", self.metadata)  # add metadata.yaml
         return str(f), None
 
+    @try_export
+    def export_mnn(self, prefix=colorstr("MNN:")):
+        """YOLOv8 MNN export using MNN https://github.com/alibaba/MNN."""
+        check_requirements("MNN")
+        import MNN  # noqa
```

Comment:
curious to ask, why use capital MNN as as package name

Reply:
The project name is [MNN](https://github.com/alibaba/MNN), just using it.